extensions/grpc/src/main/java/com/hazelcast/jet/grpc/GrpcServices.java

```diff
@@ -107,7 +112,7 @@ private GrpcServices() {
                 .withCreateContextFn(ctx -> channelFn.get().build())
                 .withCreateServiceFn((ctx, channel) -> new UnaryService<>(channel, callStubFn))
                 .withDestroyServiceFn(UnaryService::destroy)
-                .withDestroyContextFn(ManagedChannel::shutdown);
+                .withDestroyContextFn(channel -> GrpcUtil.shutdownChannel(channel, logger));
```

Comment:
it would be better to use the `logger` from ctx, you can do this by making createContextFn return `tuple2(channel, logger)` , kind of ugly, but it should be fine.